index.js

```diff
@@ -1,3 +1,5 @@
 import Link from "./src/components/link"
+import DebugMediaQueries from "./src/components/layout/debug/media-queries"
 
 export { Link }
+export { DebugMediaQueries }
```

Comment:
You can put these into one line:

```js
export { Link, DebugMediaQueries }
```

Also, make sure you're updated against master. I added SEO component and I think you'll run into an issue here.